src/network/tarcap/packets_handlers/new_dag_block_packet_handler.cpp

```diff
@@ -42,11 +40,15 @@ void DagPacketsHandler::process(const PacketData &packet_data, const std::shared
   }
 
   // Ignore new block packets when pbft syncing
-  if (syncing_state_->is_pbft_syncing()) return;
+  if (syncing_state_->is_pbft_syncing()) {
+    LOG(log_dg_) << "Ignore new dag block " << hash.abridged() << ", pbft syncing is on";
+    return;
+  }
 
   if (dag_blk_mgr_) {
     // Do not process this block in case we already have it
     if (dag_blk_mgr_->isDagBlockKnown(block.getHash())) {
+      LOG(log_dg_) << "Ignore new dag block " << hash.abridged() << ", block is already known";
       return;
     }
```

Comment:
also this check should be done before parsing transactions

Reply:
same as I said above